tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -34,7 +45,12 @@ sub get_script_run {
     handle_sp_in_settings_with_fcs("GUEST_LIST");
     my $guest_list = get_required_var("GUEST_LIST");
 
-    $pre_test_cmd = "$pre_test_cmd -p $product_upgrade -r $product_upgrade_repo -t $max_test_time -g \"$guest_list\"";
+    $pre_test_cmd = "$pre_test_cmd -p $product_upgrade -r $product_upgrade_repo -g \"$guest_list\"";
+    if (is_s390x) {
+        $pre_test_cmd .= "  | tee /tmp/s390x_guest_upgrade_test.log";
```

Comment:
I am afraid you need to add error msg into log, so sth like "2>&1 | tee /tmp/s390x_guest_upgrade_test.log " is needed.

Reply:
yes, had added the standard error info to s390x_guest_upgrade_test.log already. 
I just thought this svirt backend had covered this standard output and standard error together as enough. 
```
backend::baseclass::run_ssh_cmd
=head2 run_ssh_cmd

$ret = run_ssh_cmd($cmd [, username => ?][, password => ?][,host => ?]);
($ret, $stdout, $stderr) = run_ssh_cmd($cmd [, username => ?][, password => ?][,host => ?], wantarray => 1);
```
but, missed the important error info for s390x_guest_upgrade_test.log.
thanks so much for your great help.